docker-compose.yaml

```
@@ -37,4 +37,10 @@ services:
       MYSQL_USER: shiori
       MYSQL_PASSWORD: shiori
     ports:
-      - "3306:3306"
+      - 3306
```

Using the short form will make ports unexposed to the local environment.

```suggestion
      - "3306:3306"
```